Makefile

```diff
@@ -240,6 +245,10 @@ $(LIVE_IMG).qcow2: $(LIVE_IMG).raw | $(DIST)
 	qemu-img convert -c -f raw -O qcow2 $< $@
 	rm $<
 
+$(LIVE_IMG).rpi: $(BOOT_PART) $(EFI_PART) $(ROOTFS_IMG) $(CONFIG_IMG) | $(INSTALLER)
```

Comment:
Does this change the disk layout between RPi and other systems? Does it have a fundamentally different layout, with a different number of partitions? If so, we should document it and why it is necessary.

Reply:
It doesn't change the disk layout, but it does change how the GRUB trampoline hand-off works (see the comment right above this in the later PR @deitch). As for documenting it -- let me first ponder on how this could be generalized to other systems.

In fact, let me ponder if we shouldn't be transitioning to readconfig style of a trampoline across the board.

In the meantime -- I think it is fine for this "twist" to stay like it is.